app/views/pregnancies/_patient_information.html.erb

```diff
@@ -9,11 +9,8 @@
 
       <div class="row">
         <div class="info-form-left col-sm-6">
-          <%= f.fields_for pregnancy.patient do |p| %>
-            <%= p.text_field :secondary_person, autocomplete: 'off' %>
-            <%= p.text_field :secondary_phone, autocomplete: 'off' %>
-          <% end %>
-    
+          <strong>First and last name:</strong><br><%= pregnancy.patient.name %><br>
```

Comment:
Can you revert these changes? We want patient name and primary phone to still be editable.